packages/core/src/services/ubi/community.ts

```diff
@@ -867,56 +881,85 @@ export default class CommunityService {
+            return [{
+                user: user as AppUser,
+                isDeleted: false,
+                addedBeneficiaries: 0,
+                pending: true,
+            }];
+        } else {
+            const result = await this.manager.findAll({
+                include: [
+                    {
+                        model: this.appUser,
+                        as: 'user',
+                        include: [
+                            {
+                                model: this.appMediaContent,
+                                as: 'avatar',
+                                required: false,
+                                include: [
+                                    {
+                                        model: this.appMediaThumbnail,
+                                        as: 'thumbnails',
+                                        separate: true,
+                                    },
+                                ],
+                            },
+                        ],
+                    },
+                ],
+                where: {
+                    communityId: community.id,
+                    ...activeCondition,
                 },
-            ],
-            where: {
-                communityId: community.id,
-                ...activeCondition,
-            },
-        });
+            });
 
-        const beneficiariesAdded = await this.ubiBeneficiaryRegistry.findAll({
-            attributes: [[fn('COUNT', col('address')), 'count'], 'from'],
-            where: {
-                from: {
-                    [Op.in]: result.map((el) => el.address),
+            const beneficiariesAdded = await this.ubiBeneficiaryRegistry.findAll({
+                attributes: [[fn('COUNT', col('address')), 'count'], 'from'],
+                where: {
+                    from: {
+                        [Op.in]: result.map((el) => el.address),
+                    },
                 },
-            },
-            group: ['from'],
-            raw: true,
-        });
+                group: ['from'],
+                raw: true,
+            });
 
-        return result.map((r) => {
-            const manager = r.toJSON() as ManagerAttributes;
-            const addedBeneficiaries = beneficiariesAdded.find(
-                (el) => el.from === manager.address
-            );
-            return {
-                ...manager,
-                isDeleted: !manager.user,
-                addedBeneficiaries:
-                    Number((addedBeneficiaries as any)?.count) || 0,
-            };
-        });
+            return result.map((r) => {
+                const manager = r.toJSON() as ManagerAttributes;
+                const addedBeneficiaries = beneficiariesAdded.find(
+                    (el) => el.from === manager.address
+                );
+                return {
+                    ...manager,
+                    isDeleted: !manager.user,
+                    addedBeneficiaries:
```

Comment:
can you rename to just "added", so it goes aligned with the subgraph https://github.com/impactMarket/subgraph/blob/main/schema.graphql#L60
you can rename it, I think these values aren't being used yet.

Reply:
done